### CS Homework: Week 2

#### 2.2.7:

In [2]:
2**40

1099511627776

In [3]:
8*2**20

8388608

In [4]:
1099511627776/8388608

131072.0

The hard drive can approximately hold 131072 songs

#### 2.2.3:

In [5]:
15 * 3 - 2

43

The equation above equals 43 because you multiple 15 and 3 then subtract 2.

In [6]:
15-3*2

9

The equation above equals 9 because you multiple 3 and 2 then subtract 15.

In [7]:
15*3//2

22

The equation above equals 22 because you multiple 15 and 3 then divide by 2 to get the approximate answer.

In [8]:
15*3/2

22.5

The equation above equals 22.5 because you multiple 15 and 3 then divide by 2.

In [9]:
15*3 % 2

1

The equation above equals 1 because when you are using the mod operator you will get a remainder and in this problem, you first multiple 15 and 3 then divide your answer by 2 to receive a remainder.

In [10]:
15*3/2e0

22.5

The equation above equals 22.5 because you are using a floating point number. A floating point number is anything written in scientific notation.

#### 2.3.4:

The equal sign (=) is called the assignment operator because it is used to assign values to names. The reason why you can not switch the values of x and y is because when it is written:

x=y you are saying x is assigned to y and then when, y=x you are saying y is assigned to x

#### Last question:

In [1]:
myEmployees = ["Ali", "Ben", "Charles", "Deb", "Ellie", "Fred", "Greg", "Henry", "Isaac", "Jack"]

daysOpen = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday"]

daystext = input("Please input days open as a list")
employeetext = input("Hi Boss Please input my employees as a list")

daysOpen = daystext.split(", ")
myEmployees = employeetext.split(", ")

ScheduleList = []

for i in range(len(daysOpen)):
    ScheduleList.append([])
    
for employee in myEmployees:
    dayIndex = myEmployees.index(employee)%len(daysOpen)
    ScheduleList[dayIndex].append(employee)

print(ScheduleList)


Please input days open as a list monday tuesday wednesday
Hi Boss Please input my employees as a list ali bob jake jessie kyle hannah may


[['ali bob jake jessie kyle hannah may']]
